google-cloud-storage/lib/google/cloud/storage/bucket.rb

```diff
@@ -898,10 +898,6 @@ def default_acl
         # @see https://cloud.google.com/storage/docs/json_api/v1/buckets/setIamPolicy
         #   Buckets: setIamPolicy
         #
-        # @param [Boolean] force Force load the latest policy when `true`.
```

Comment:
I do not want this parameter removed from the documentation. There are valid use cases for using it.